bedrock/firefox/templates/firefox/messaging-experiment/privacy_by_default.html

```diff
@@ -0,0 +1,74 @@
+  
+  {% extends "firefox/messaging-experiment/base.html" %}
+  
+  {# L10n: HTML page title #}
+  {% block page_title %}{{ _('Firefox: Private By Default') }}{% endblock %}
+
+  {% set _entrypoint = 'mozilla.org-privacy-messaging-experiment' %}
+  {% set _utm_campaign = 'privacy-by-default' %}
+  
+  {% block body_class %}{{ super() }} privacy-default{% endblock %}
+  
+  {% block content_intro %}
+    {% call hero(
+      title=_('Firefox: Private By Default'),
+      desc=_("Firefox automatically protects your privacy without sacrificing performance. So go ahead and browse with confidence, knowing Firefox is protecting what you do online."),
+      class='mzp-t-firefox mzp-t-dark',
+      include_cta=True,
+      heading_level=1
+    ) %}
+  
+    <p class="primary-cta">
+      <div>
+        <a href="{{ url('firefox.privacy.index') }}" class="mzp-c-button mzp-t-product" data-cta-text="Learn About Firefox Privacy" data-cta-type="button">{{ _('Learn About Firefox Privacy') }}</a>
+      </div>
+    </p>
+    {% endcall %}
+  {% endblock %}
+  
+
+  {% block content_primary %}
+    <div class="body-primary">
+      <div class="c-picto-block t-adjacent-image">
+        <div class="c-picto-block-image">
+          <img src="{{ static('img/icons/hide_password-orange.svg') }}" alt="">
+        </div>
+        <h3 class="c-picto-block-title">Your privacy is our first thought, not an afterthought</h3>
+        <div class="c-picto-block-body">
+          <p>{{ _('
+            Tracking has become an epidemic online: companies follow every move, click and purchase, collecting data to predict and influence what you’ll do next. That’s why Firefox mobile and desktop browsers have Enhanced Tracking Protection on by default.') }}</p>
+        </div>
+      </div>
+
+      <div class="c-picto-block t-adjacent-image">
+        <div class="c-picto-block-image">
+          <img src="{{ static('img/icons/laptop_warning-pink.svg') }}" alt="">
+        </div>
+        <h3 class="c-picto-block-title">You can worry less about malware</h3>
+        <div class="c-picto-block-body">
+          <p>{{ _('Firefox automatically stops malware, including some types that can use your system’s computing power, slowing your computer and even raising your energy bills.') }}</p>
+        </div>
+      </div>
+
+      <div class="c-picto-block t-adjacent-image">
+        <div class="c-picto-block-image">
+          <img src="{{ static('img/icons/cryptomining-purple.svg') }}" alt="">
+        </div>
+
+        <h3 class="c-picto-block-title">Cryptominers stopped in their tracks</h3>
+        <div class="c-picto-block-body">
+          <p>{{ _('Cryptominers are a type of malware that use your system’s computing power to mine digital money. Cryptominers are blocked by default with Firefox’s Standard settings.') }}</p>
+        </div>
+      </div>
+    </div>
+  {% endblock %}
+  
+  {% block js %}
+    {{ js_bundle('default_privacy_messaging') }}
```

Comment:
Since we're no longer linking to about:protections we should be able to delete this bundle.